gradle.properties

```diff
@@ -1,2 +1,2 @@
 group=com.sk89q.worldguard
-version=7.1.0-SNAPSHOT
+version=7.1.0-SNAPSHOT-Custom-Folia
```

Comment:
For full folia support we should not change the version

Reply:
This has been resolved and updated.